.github/workflows/windows.yml

```diff
@@ -284,53 +286,25 @@ jobs:
             Expand-Archive openvino_tests.zip -DestinationPath "${{ env.INSTALL_TEST_DIR }}"
           popd
 
-      - name: Check extraction
-        run: |
-          ls "${{ github.workspace }}"
-          ls "${{ env.INSTALL_DIR }}"
-          ls "${{ env.INSTALL_TEST_DIR }}"
-
       - name: Clone OpenVINO
         uses: actions/checkout@v4
         with:
           path: 'openvino'
 
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: ${{ env.PYTHON_VERSION }}
 
-      - name: Install python dependencies
+      - name: Install Python API tests dependencies
         run: |
-          # For Python API
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/wheel/requirements-dev.txt
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/requirements.txt
-
-          # For running Python API tests
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/src/compatibility/openvino/requirements-dev.txt
+          # For torchvision to OpenVINO preprocessing converter
+          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/src/openvino/preprocess/torchvision/requirements.txt
 
-          # For running ONNX frontend unit tests
-          python3 -m pip install --force-reinstall -r ${{ env.OPENVINO_REPO }}/src/frontends/onnx/tests/requirements.txt
-
-          # For running TensorFlow frontend unit tests
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/frontends/tensorflow/tests/requirements.txt
-
-          # For running Paddle frontend unit tests
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/frontends/paddle/tests/requirements.txt
-
-          python3 -m pip install -r ${{ env.LAYER_TESTS_INSTALL_DIR }}/requirements.txt
+          # TODO: replace with Python API tests requirements
+          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_dev.txt
 
-      - name: Install MO dependencies
-        run: |
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_mxnet.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_caffe.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_kaldi.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_onnx.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_tf2.txt `
-          -r ${{ env.OPENVINO_REPO }}/tools/mo/requirements_dev.txt
-
-      - name: Install Python wheels
-        run: |
-          python3 -m pip install openvino-dev --force-reinstall --find-links=${{ env.INSTALL_DIR }}\tools
+      - name: Install OpenVINO Python wheels
+        run: python3 -m pip install openvino-dev[mxnet,caffe,kaldi,onnx,tensorflow2] --find-links=${{ env.INSTALL_DIR }}\tools
```

Comment:
better to replace with approach like we have in macOS pipeline to explicitly install via file name

we can do it separately for all other pipelines.